src/__test__/handle-data.spec.ts

```diff
@@ -267,9 +268,27 @@ describe('handle-data', () => {
 
   describe('getTotalPosition:', () => {
     let expectedPosition: TotalsPosition;
-
+    let viewService: ViewService;
+    
     beforeEach(() => {
       expectedPosition = { atBottom: false, atTop: false };
+      viewService = { viewState: { skipTotals: true } as ViewState } as ViewService;
+    });
+
+    describe('When viewService?.viewState?.skipTotals is true', () => {
```

Comment:
I would say you only need one test for this.

Reply:
I think these tests should be one where skipTotals is false and one where it is true